samples/Common/cs/FrameReaderFrameSource.cs

```diff
@@ -168,12 +168,12 @@ private async Task InitializeMediaFrameSourceAsync()
             int preferredFrameWidth = 1920;
             if (m_desiredImageDescriptor != null && m_desiredImageDescriptor.Width != -1)
             {
-                preferredFrameWidth = m_desiredImageDescriptor.Width;
+                preferredFrameWidth = Math.Abs(m_desiredImageDescriptor.Width);
```

Comment:
>Math.Abs [](start = 38, length = 8)

What is the meaning for a width = -640?

Reply:
a width divisible by 640

Reply:
that is in the context of a SkillFeatureImageDescriptor.Width, in the context of this app consuming such object, we just look for the closest available resolution to the abs() of this value, I could change that logic to accommodate negative values properly